core/package-graph/index.js

```diff
@@ -190,7 +299,10 @@ class PackageGraph extends Map {
   }
 
   /**
-   * Return a tuple of cycle paths and nodes, which have been removed from the graph.
+   * Return a tuple of cycle paths and nodes.
+   *
+   * @deprecated Use collapseCycles instead.
```

Comment:
I couldn't change the signature of this method for backward compatibility. Instead of adding a new boolean switch, I decided to create a new method. The old and new `visits` are similar but incompatible.